src/scss/shared/_controls.scss

```diff
@@ -0,0 +1,74 @@
+/// @access private
+/// @output Shared base styles for checkboxes, toggles, round- and box-style radio buttons.
+@mixin _oFormsControlsBase($disabled: null) {
+	position: absolute;
+	padding: 0;
+	width: 24px;
+	height: 24px;
+
+	&:checked + span:after {
+		opacity: 1;
+	}
+
+	@if $disabled {
+		&:disabled {
+			opacity: 0;
+
+			+ span {
+				opacity: 0.4;
+			}
+		}
+	}
+}
+
+/// @access private
+/// @output Shared pseudo element styles for checkboxes, toggle, round- and box-style radio buttons.
+@mixin _oFormsControlsPseudoElements {
+	content: '';
+	cursor: pointer;
```

Comment:
Looks like a pointer used to be applied to the labels of checkbox and radio buttons too.